modules/apps/transfer/types/trace.go

```diff
@@ -149,26 +151,20 @@ func (dt DenomTrace) Validate() error {
 //   - Unprefixed denomination: 'baseDenom'
 //
 // 'baseDenom' may or may not contain '/'s
-func ValidatePrefixedDenom(denom string) error {
-	denomSplit := strings.Split(denom, "/")
-	if denomSplit[0] == denom && strings.TrimSpace(denom) != "" {
-		// NOTE: no base denomination validation
-		return nil
-	}
-
-	pathSlice, baseDenom := extractPathAndBaseFromFullDenom(denomSplit)
-	if strings.TrimSpace(baseDenom) == "" {
+func ValidatePrefixedDenom(fullPath string) error {
```

Comment:
I know this has already been like this for a while, but we also use `PrefixedDenom` and `FullPath` for trace + base denom. It would also be nice to converge now into a single term. I don't particularly like either, but what about `IBCDenom` (if we use on-chain denom for `ibc/<hash>`) or `ICS20Denom`?

Reply:
I think it's good to open an issue and take discussion there. I'm happy to see the function renamed

Reply:
this function will be deleted in a future pr